app.js

```diff
@@ -22,4 +22,20 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+  .then((beerAll => {
+    //const data = { beerAll: oneBeer[0]} 
+    console.log(beerAll)
+    res.render('beers',{beerAll: beerAll});
+  })).catch((error) => console.log(error))
+});
+app.get('/random-beer' ,(req, res) => {
+  punkAPI.getRandom()
+  .then((beer1) => {
+    console.log(beer1)
+    res.render('random-beer',{beer1: beer1});
```

Comment:
Here I would pass just the element, instead of the whole array, that way you could access the properties directly instead of using the `#each`.

I wouldn't call the property beer1, maybe oneBeer or randomBeer could be options

```suggestion
    res.render('random-beer',{ randomBeer: beer1[0] } );
```